skbio/stats/composition.py

```diff
@@ -608,10 +612,274 @@ def centralize(mat):
 
     """
     mat = closure(mat)
-    cen = ss.gmean(mat, axis=0)
+    cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
 
+@experimental(as_of="0.4.0-dev")
+def ancom(matrix, categories,
+          alpha=0.05,
+          tau=0.02,
+          theta=0.1,
+          multiple_comparisons_correction=None,
+          significance_test=scipy.stats.ttest_ind):
+    r"""
+    Calculates pairwise log ratios between all otus
+    and performs a signficance test to determine if there is a
+    significant difference in feature ratios with respect to the
+    variable of interest
+
+    In an experiment with only two treatments, this test tests the
+    following hypothesis for feature :math:`i`
+    :math:`H_{0i}:\E[\ln(u_i^{(1)})] = \E[\ln(u_i^{(2)})]`
+
+    where :math:`u_i^{(1)}` is the mean abundance for feature
+    :math:`i` in the first group and :math:`u_i^{(2)}` is the
+    mean abundance for feature :math:`i` in the second group.
+
+    This method can be extended two an arbitrary number of classes
+    by passing in a different statistical function.
+
+
+    Parameters
+    ----------
+    matrix: pd.DataFrame or array_like
```

Comment:
Originally it was meant to operate on relative abundances.

But it shouldn't matter if the input were counts or normalized counts - they'll yield the same result.